<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>KisWdgIndexColors</class>
 <widget class="QWidget" name="KisWdgIndexColors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>601</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item column="0" row="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ramps</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item column="0" row="1">
       <widget class="QWidget" name="colorsBox" native="true">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <layout class="QGridLayout" name="layoutColors">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
        </layout>
       </widget>
      </item>
      <item column="0" row="2">
       <widget class="QCheckBox" name="diagCheck">
        <property name="text">
         <string>Diagonal Gradients</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item column="0" row="2">
    <widget class="QGroupBox" name="gradientStepsBox">
     <property name="title">
      <string>Gradient Steps</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="layoutGradientSteps">
      <item column="0" row="3">
       <widget class="QCheckBox" name="colorLimitCheck">
        <property name="text">
         <string>Limit to</string>
        </property>
       </widget>
      </item>
      <item column="1" row="3">
       <widget class="KisIntParseSpinBox" name="colorLimit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>256</number>
        </property>
        <property name="value">
         <number>31</number>
        </property>
       </widget>
      </item>
      <item column="1" row="0">
       <widget class="KisIntParseSpinBox" name="inbetweenSpinBox">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item column="0" row="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>In-between ramps</string>
        </property>
       </widget>
      </item>
      <item colspan="2" column="0" row="1">
       <layout class="QGridLayout" name="layoutRowSteps">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item column="0" row="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Color count:</string>
        </property>
       </widget>
      </item>
      <item column="1" row="2">
       <widget class="QLabel" name="colorCount">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item column="0" row="3">
    <widget class="QGroupBox" name="indexingBox">
     <property name="title">
      <string>Indexing Factors</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item column="0" row="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>L*</string>
        </property>
       </widget>
      </item>
      <item column="1" row="0">
       <widget class="QSlider" name="luminanceSlider">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item column="1" row="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item column="0" row="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>a*</string>
        </property>
       </widget>
      </item>
      <item column="1" row="2">
       <widget class="QSlider" name="aSlider">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item column="0" row="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>b*</string>
        </property>
       </widget>
      </item>
      <item column="1" row="3">
       <widget class="QSlider" name="bSlider">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item column="0" row="4">
    <widget class="QGroupBox" name="otherBox">
     <property name="title">
      <string>Other</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item column="0" row="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Alpha Steps</string>
        </property>
       </widget>
      </item>
      <item column="1" row="0">
       <widget class="KisIntParseSpinBox" name="alphaStepsSpinBox">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>32</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisIntParseSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kis_int_parse_spin_box.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>colorLimitCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorLimit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>